tests/cpp/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2013-2015, EPFL/Blue Brain Project
 #                     Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 #                     Raphael Dumusc <raphael.dumusc@epfl.ch>
+#
+# Change this number when adding tests to force a CMake run: 0
```

Comment:
But your are forced to run cmake, because git pull && make would not get that new file.